shared/utils.ts

```diff
@@ -42,6 +42,7 @@ export const smartTrim = (text, maxLength = 200) => {
   }
 };
 
+// Todo: We need to allow for subdomain validation for the telgram bot
```

Comment:
Resolve this, so we can keep the validURL! Or make a new regex just for TG.

Reply:
You mean resolve the conversation of resolve the issue. If so, what do I do to solve this?

Reply:
Resolve the issue by ensuring that the telegram URL you provided passes the regex...